### ISSUE TITLE
Can't manage plugins

### ISSUE THREAD

**Issue by yunionlassalle** [cerebroapp/cerebro#457]
- [ ] I am on the [latest](https://github.com/KELiON/cerebro/releases/latest) Cerebro.app version
- [x ] I have searched the [issues](https://github.com/KELiON/cerebro/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: 
- **Cerebro.app version**: 
- **Relevant information from devtools** _(See above how to open it)_: 

## Issue

Hi I'm using Linux Mint 18.2 Sonya and have a lot of issues with Cerebro. Haved installed the last version but doesn't work, have right now the 0.3.1 version. My main problem is that meny plugins doesnt work either, and I cant even see the installed plugins, when I go to the plugins options just get the search results. :( I have no idea of what to do

**Comment by AlexeyKot** [comment]
I have the same issue, Linux Mint 18.3 and Cinnamon. When I start typing "Plugins", I see "Manage Plugins" icon appear,when I click on it - I see "plugins" word with google icon. There's no way to manage plugins.

**Comment by KELiON** [comment]
@yunion @AlexeyKot can you attach logs from developer tools?

**Comment by carlos-wong** [comment]
@AlexeyKot I think you need use `TAB` on `Manage Plugins`. It will show the installed and developing plugins

**Comment by MynockSpit** [comment]
The search term `plugins ` triggers this error. Will dig a bit deeper...

```
Uncaught (in promise) TypeError: Cannot read property 'replace' of undefined
    at Object.o (main.bundle.js:1)
    at Object.c (main.bundle.js:1)
    at S.t (main.bundle.js:1)
    at h (main.bundle.js:1)
    at Function.ii (main.bundle.js:1)
    at t (main.bundle.js:1)
    at ei (main.bundle.js:1)
    at main.bundle.js:1
    at m (main.bundle.js:1)
    at Sn (main.bundle.js:1)
o @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
S.t @ main.bundle.js:1
h @ main.bundle.js:1
ii @ main.bundle.js:1
t @ main.bundle.js:1
ei @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
m @ main.bundle.js:1
Sn @ main.bundle.js:1
Oe.value @ main.bundle.js:1
Sn @ main.bundle.js:1
n.toJSON.n.valueOf.n.value @ main.bundle.js:1
(anonymous) @ main.bundle.js:1
```

**Comment by MynockSpit** [comment]
Dug deeper. This is fixed in 3.2. This is only still an issue b/c https://cerebroapp.com/ downloads 3.0 by default. Can be closed.

**Comment by ShinobiWPS** [comment]
I confirm it is fixed in 0.3.2!

**Comment by AustinGil** [comment]
How do you install the newer releases on Windows? I see the releases page has a 0.4.0 version. The readme mentions where to download and how to use it after install, but it skips any install steps.

**Comment by dubisdev** [comment]
HI @AustinGil we are working in generating releases
Last can be found [here](https://github.com/cerebroapp/cerebro/releases)

I'm closing this as it seems to be fixed in v0.3.2
